PlayerUI/Views/PUIPlayerView.swift

```diff
@@ -1020,14 +1020,14 @@ public final class PUIPlayerView: NSView {
 
         guard let durationTime = player.currentItem?.duration else { return }
 
-        let modifier = CMTimeMakeWithSeconds(seconds, durationTime.timescale)
+        let modifier = CMTimeMakeWithSeconds(seconds, preferredTimescale: durationTime.timescale)
```

Comment:
Slightly off-topic, but I would use the modern constructor instead:
[`CMTime(seconds: Double, preferredTimescale: CMTimeScale)`](https://developer.apple.com/documentation/coremedia/cmtime/1489786-init)